transport-native-unix-common/src/main/java/io/netty/channel/unix/Limits.java

```diff
@@ -21,11 +21,22 @@
 import static io.netty.channel.unix.LimitsStaticallyReferencedJniMethods.uioMaxIov;
 
 public final class Limits {
-    public static final int IOV_MAX = iovMax();
```

Comment:
motivation:
It is very difficult to static final native method for unit test. so, I modified the static final fields in Limits to static method.

there is an unit test in io.netty.channel.unix.tests.UnixChannelUtilTest.  After modiified, it becomes very easy to do unit test, and it looks like  same efficiency as before.

I mentioned it in commit log. I confused this log doesn't show on this pull request. Maybe it is  I am not familiar with the use of github.